cadc-file-validate/README.md

```diff
@@ -0,0 +1,74 @@
+# cadc-file-validate 0.1
+
+Command line tool to ensure validity of the information stored in the Inventory Database and what is served from a Storage Adapter.  
+
+This is intended to run periodically at any site to determine what changes, if any, are to be made to either the Storage Inventory database, or
+the storage system.
+
+## Configuration
+
+A file called `cadc-file-validate.properties` should be available from the `/config` directory.  All properties in the file are loaded and set
+as System properties:
+
+```java
+Properties properties = new Properties();
+properties.load("/config/cadc-file-validate.properties");
+System.setProperties(properties);
+```
+
+### Sample configuration (`cadc-file-validate.properties`)
+
+```
+# Used to set the bucket range to validate
+org.opencadc.inventory.validate.bucket.start = {bucket-range-start}
+org.opencadc.inventory.validate.bucket.end = {bucket-range-end}
+
+#######
+## Storage Adapter Inventory settings
+#######
+
+# The storage adapter to use for storage.
+# Consult the storage adapter instructions for adapter-specific configuration.  The library MUST be
+# accessible from this application's classpath.
+#org.opencadc.inventory.storage.StorageAdapter={fully-qualified-classname of implementation}
+org.opencadc.inventory.storage.StorageAdapter=org.opencadc.inventory.storage.fs.FileSystemStorageAdapter
+
+# The SQL generator implementation (default shown)
+#org.opencadc.inventory.db.SQLGenerator=org.opencadc.inventory.db.SQLGenerator
+
+# The schema to use
+org.opencadc.inventory.db.schema=inventory
```

Comment:
The database connection will require schema, username, password, and jdbc url.

Should also be verbose and debug log level option...

The packages to set to those levels (as opposed to default warn) could be listed (single prop with space-separated list of package names) or the packages could be hard coded.... but debug on all of org.opencadc and ca.nrc.cadc is going to be really noisy